### ISSUE TITLE
likify doesn't work when go to pull request page from pull request list page

### ISSUE THREAD

**Issue by methane** [berkerpeksag/cpython-bpo-linkify#7]
To reproduce:

1. Open https://github.com/python/cpython/pulls
2. Click any pull request which has bpo-XXXXX prefix.
3. bpo-XXXXX is not linkified.

**Comment by berkerpeksag** [comment]
+1, would you like to send a PR?

**Comment by methane** [comment]
I tried to implement, but I gave it up.
Since github is (partially) using "Single Page Application", it's difficult than I thought.

**Comment by berkerpeksag** [comment]
Yeah, there are edge cases where the extension doesn't work very well yet. SPA problem may be solved by listening some events or doing periodic checks on github.com, but I haven't had a chance to try it yet.
